scopes/ui-foundation/ui/webpack/webpack.base.config.ts

```diff
@@ -49,6 +39,11 @@ export default function createWebpackConfig(
   // passed into alias object. Uses a flag if passed into the build command
   const isEnvProductionProfile = process.argv.includes('--profile');
 
+  const { styleLoaders, stylePlugins } = makeStyleLoaders({
+    postcssOptions: postCssConfig,
+    styleInjector: isEnvProduction ? undefined : 'style-loader',
```

Comment:
what about the miniCssOptions for the plugin? you changed the behaviour of the plugin now

Reply:
passed as injectorOptins